src/controllers/productController.js

```diff
@@ -0,0 +1,20 @@
+import _ from 'underscore';
```

Comment:
I'd recommend using lodash instead. Or Array methods (find and filter are already there).